src/index.js

```diff
@@ -200,7 +201,19 @@ class TrustWeb3Provider extends EventEmitter {
   }
 
   eth_sign(payload) {
```

Comment:
Should we have duplicates with eth_ prefix for all sign call

Reply:
`signPersonalMessage` => add eth msg prefix
`signMessage` => sign directly